packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -734,7 +732,12 @@ export class DocumentDeltaConnection
 		const errorObj = createGenericNetworkError(
 			`socket.io (${handler}): ${message}`,
 			{ canRetry },
-			{ driverVersion },
+			{
+				driverVersion,
+				details: JSON.stringify({
+					...this.getConnectionDetailsProps(),
```

Comment:
not sure why we JSON here, but do not do it in other places. I'd keep it without JSON

Reply:
don't we always add the details with JSON.stringfy ? At least on the previous occurrences in this file

Reply:
For Data_details, we always stringify.